spec/models/event_spec.rb

```diff
@@ -3,4 +3,18 @@
 
 RSpec.describe Event, type: :model do
   it { is_expected.to have_many(:ticketholders).through(:tickets).source(:supporter) }
+  it { is_expected.to define_enum_for(:in_person_or_virtual).with_values(in_person: "in_person", virtual: "virtual").backed_by_column_of_type(:string).validating }
```

Comment:
Can also use `itself` here as I suggested with the enum above.
```suggestion
  it { is_expected.to define_enum_for(:in_person_or_virtual).with_values(%w[in_person virtual].index_by(&:itself)).backed_by_column_of_type(:string).validating }
```